<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>785</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pneumothorax CAD</string>
  </property>
  <widget class="QPushButton" name="load_image">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Load CXR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="predict_image">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>30</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Predict CXR</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="oriCXR">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>711</width>
     <height>671</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="result">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>40</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
